frontend/public/components/app.jsx

```diff
@@ -163,6 +163,8 @@ class App_ extends React.PureComponent {
           <div id="app-content" className="co-m-app__content">
             <ConsoleNotifier location="BannerTop" />
             <Page
+              // Need to pass mainTabIndex=null to enable keyboard scrolling as default tabIndex is set to -1 by patternfly
+              mainTabIndex={null}
```

Comment:
With this addition, we should also be able to remove the `tabIndex` on the sidebar. 
 See https://github.com/openshift/console/blob/master/frontend/public/components/sidebars/resource-sidebar.tsx#L25-L26

Reply:
Done.